components/stacks-network/src/orchestrator.rs

```diff
@@ -2266,63 +2251,73 @@ events_keys = ["*"]
                 .kill_container(bitcoin_explorer_container_id, options.clone())
                 .await;
             ctx.try_log(|logger| slog::info!(logger, "Terminating bitcoin-explorer"));
-            let _ = docker.remove_container(bitcoin_explorer_container_id, None);
+            let _ = docker
+                .remove_container(bitcoin_explorer_container_id, None)
+                .await;
         }
 
         if let Some(ref stacks_explorer_container_id) = self.stacks_explorer_container_id {
             let _ = docker
                 .kill_container(stacks_explorer_container_id, options.clone())
                 .await;
             ctx.try_log(|logger| slog::info!(logger, "Terminating stacks-explorer"));
-            let _ = docker.remove_container(stacks_explorer_container_id, None);
+            let _ = docker
+                .remove_container(stacks_explorer_container_id, None)
+                .await;
         }
 
         if let Some(ref bitcoin_node_container_id) = self.bitcoin_node_container_id {
             let _ = docker
                 .kill_container(bitcoin_node_container_id, options.clone())
                 .await;
             ctx.try_log(|logger| slog::info!(logger, "Terminating bitcoin-node"));
-            let _ = docker.remove_container(bitcoin_node_container_id, None);
+            let _ = docker
+                .remove_container(bitcoin_node_container_id, None)
+                .await;
```

Comment:
The await here and the lines below could fix the case where a container isn't properly removed at the end of `clarinet integrate` (leading to conflict when running integrate again)